industrial_ci/src/docker.sh

```diff
@@ -162,7 +162,11 @@ function ici_build_default_docker_image() {
   ici_docker_build - <<EOF > /dev/null
 FROM ubuntu:$UBUNTU_OS_CODE_NAME
 
-RUN apt-get update -qq && apt-get -qq install --no-install-recommends -y wget ca-certificates sudo
+RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
```

Comment:
From the [debconf manpage](http://manpages.ubuntu.com/manpages/xenial/man7/debconf.7.html):

>   You  can  change  the  default  frontend  debconf uses by reconfiguring
       debconf. On the other hand, if you just want to change the frontend for
       a  minute,  you can set the DEBIAN_FRONTEND environment variable to the
       name of the frontend to use.